frontend/stores/organization.ts

```diff
@@ -96,96 +92,32 @@ export const useOrganizationStore = defineStore("organization", {
     async fetchById(id: string | undefined) {
       this.loading = true;
 
-      const [responseOrg, responseOrgTexts, responseOrgEvents] =
-        await Promise.all([
-          useAsyncData(
-            async () =>
-              await fetchWithoutToken(`/entities/organizations/${id}`, {})
-          ),
-          useAsyncData(
-            async () =>
-              await fetchWithoutToken(
-                `/entities/organization_texts?org_id=${id}`,
-                {}
-              )
-          ),
-          useAsyncData(
-            async () =>
-              await fetchWithoutToken(
-                `/entities/organization_events?org_id=${id}`,
-                {}
-              )
-          ),
-          // useAsyncData(
-          //   async () =>
-          //     await fetchWithoutToken(
-          //       `/entities/organization_faq?org_id=${id}`,
-          //       {}
-          //     )
-          // ),
-          // useAsyncData(
-          //   async () =>
-          //     await fetchWithoutToken(
-          //       `/entities/organization_resources?org_id=${id}`,
-          //       {}
-          //     )
-          // ),
-        ]);
+      const [responseOrg] = await Promise.all([
+        useAsyncData(
+          async () =>
+            await fetchWithoutToken(`/entities/organizations/${id}`, {})
+        ),
+      ]);
```

Comment:
`Promise.all()` is no longer necessary since there is only one async request

Reply:
Thanks for pointing this out, @cquinn540!